### PR TITLE
Readme: Update readme for debian users

### PR DESCRIPTION
**Pull request by JasonMo1**
Yeah, another commit for debian users. There are some difference between debian and ubuntu, right?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -89,25 +89,58 @@ The OS is designed to work with an MMU, thus, the target must have 4 swappable v
...
+Some commands are different between linux and windows, so we can't give you an example to build zos on windows, if you have any progress on it, please contact us.
+
 On Ubuntu, the following commands can be used to install the dependencies. They must be run as a user, not root!
 ```
 sudo apt update
 sudo apt install git python3 python3-pip
 pip3 install --ignore-installed --user kconfiglib
 ```
 
+On Debian, run this:
```
Comment (Zeal8bit) [inline README.md:109]:
```suggestion
On Debian, the following commands can be used instead:
```

File: README.md [hunk 2]
```diff
@@ -89,25 +89,58 @@ The OS is designed to work with an MMU, thus, the target must have 4 swappable v
 
 ## Requirements
 
-At the moment, the project has only been assembled on Linux (Ubuntu 20.04 and 22.04), it should be compatible with Mac OS and Windows as long as you have:
+At the moment, the project has only been assembled on Linux (Ubuntu 20.04 and 22.04, Debian 12(bookworm)), it should be compatible with Mac OS and Windows as long as you have:
```
Comment (Zeal8bit) [inline README.md:92]:
```suggestion
At the moment, the project has only been assembled and tested on Linux (Ubuntu 20.04 and 22.04, Debian 12 Bookworm), it should be compatible with Mac OS and Windows as long as you have:
```

File: README.md [hunk 3]
```diff
@@ -89,25 +89,58 @@ The OS is designed to work with an MMU, thus, the target must have 4 swappable v
...
+```
+sudo apt update
+sudo apt install git python3
+sudo apt install python3-kconfiglib
+```
+
 For installing Z88DK, please [check out their Github project](https://github.com/z88dk/z88dk).
 
-## Configuring Zeal 8-bit OS
+If you got error like this:
```
Comment (Zeal8bit) [inline README.md:118]: This isn't necessary, it is an issue related to z88dk installation more than Zeal 8-bit OS configuration

File: README.md [hunk 4]
```diff
@@ -89,25 +89,58 @@ The OS is designed to work with an MMU, thus, the target must have 4 swappable v
...
-At the moment, the project has only been assembled on Linux (Ubuntu 20.04 and 22.04), it should be compatible with Mac OS and Windows as long as you have:
+At the moment, the project has only been assembled on Linux (Ubuntu 20.04 and 22.04, Debian 12(bookworm)), it should be compatible with Mac OS and Windows as long as you have:
 
 * bash
 * git (to clone this repo)
 * make
 * python3 with pip3. Used for the `menuconfig`.
 * z88dk v2.2 (or later). Only its assembler, `z80asm`, is strictly required. The latest version of `z80asm` must be used as earlier versions don't have support for `MACRO`.
 
+Some commands are different between linux and windows, so we can't give you an example to build zos on windows, if you have any progress on it, please contact us.
```
Comment (Zeal8bit) [inline README.md:100]: This isn't necessary as we stated we only tested on Windows at the moment. Some users reported that it assembles well using WSL, so in that case, it is not very different from Linux. 

Anyway, a Windows section in the future will be added

Comment (JasonMo1) [reply README.md:100]: Actually, I use WSL to build zos too, And I have try to build zos at another Subsystem. Windows here means a environment without a Subsystem like WSL, Msys, cygwin and so on. There is makefile support for windows, but The commands should be powershell-styled.

Comment (JasonMo1) [reply README.md:100]: We should add a Makefile.bat or add system prompt in Makefile.

**Pull request closed without merging**